src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            String stringFromFile = bufferedReader.readLine();
+            while (stringFromFile != null) {
+                String[] info = stringFromFile.split(",");
+                if (info[0].equals("supply")) {
+                    supply += Integer.parseInt(info[1]);
+                }
+                if (info[0].equals("buy")) {
+                    buy += Integer.parseInt(info[1]);
+                }
+                stringFromFile = bufferedReader.readLine();
+            }
+
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Error catch!");
+        } catch (IOException e) {
+            throw new RuntimeException("Error catch!");
```

Comment:
let's make exception messages more informative and add `e` as a cause (second parameter)
example is above